packages/models-library/src/models_library/user_preferences.py

```diff
@@ -0,0 +1,177 @@
+from collections.abc import Mapping
+from enum import auto
+from typing import Any, ClassVar, TypeAlias
+
+from pydantic import BaseModel, Field
+from pydantic.main import ModelMetaclass
+
+from .services import ServiceKey
+from .utils.enums import StrAutoEnum
+
+IntStr: TypeAlias = int | str
```

Comment:
```suggestion
IntOrStr: TypeAlias = int | str
```

Reply:
actually I've removed these. I've solved it differently via base types, without having to redefine/import them